tools/docker/start.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd web-platform-tests
+git pull --depth=1
+
+sudo sh -c './wpt make-hosts-file >> /etc/hosts'
+
+# Install Chome dev
+deb_archive=google-chrome-unstable_current_amd64.deb
+wget https://dl.google.com/linux/direct/$deb_archive
+
+sudo gdebi -n $deb_archive
```

Comment:
> Note: Installing Google Chrome will add the Google repository so your system
> will automatically keep Google Chrome up to date. If you don’t want Google's
> repository, do `sudo touch /etc/default/google-chrome` before installing the
> package.

Source: https://www.google.com/intl/en/chrome/browser/

Is this relevant given the expected lifespan of the container?

Reply:
No; any changes only persist for the lifetime of the container, which is just for a single test run.